### PR TITLE
Add static backlog and links to issue board

### PR DESCRIPTION
**Pull request by michaelfedell**

### CODE REVIEW

File: CHARTER.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+This project was started as a way to improve the grocery shopping experience, especially for young people who are still learning to cook. The project also serves as a testing ground for various tools and technologies such as cloud computing, machine-learning driven recommendation systems, application deployment and maintenance, as well as basic web development in the Flask-python ecosystem.
+
+## Vision
+
+To create a simple and fun experience out of grocery shopping which inspires creativity and exploration.
+
+## Mission
+
+Learn from experienced cooks and grocery shoppers and provide the resulting guidance in a sleek application that makes it easy for users to manage grocery lists, find new recipe ideas, and share their findings with friends.
```
Comment (finnqiao) [inline CHARTER.md]: Most of relevant features that were listed below are all in the Icebox, perhaps better to narrow scope of Mission.

File: CHARTER.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+## Success Criteria
+
+- [] 30% utilization of all served recommendations
+- [] 75% of users adopt recommendations
+- [] ≥ 80% positive feedback on UX design
+- [] manage grocery lists for at least 20 users
```
Comment (finnqiao) [inline CHARTER.md]: 20 users out of how many? I think the previous points already encapsulate usage amount.

File: CHARTER.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+## Planned Work
+
+[Click here](https://github.com/michaelfedell/smart_cart/issues) for the live issue board
+
+### Data Management
+
+Backlog:
+
+- Preliminary EDA
```
Comment (finnqiao) [inline CHARTER.md]: A lot of what else is in here can be considered under preliminary EDA, maybe reword as something else.

File: CHARTER.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+- Joins/Unions
+- Text Cleaning
+- Move from CSV to RDB
```
Comment (finnqiao) [inline CHARTER.md]: What is the reason for this?

File: CHARTER.md [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+
+Icebox:
+
+- Fuzzy Matching
+
+### Rec System
```
Comment (finnqiao) [inline CHARTER.md]: Perhaps clearer: Rec system for ____

File: CHARTER.md [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+
+Backlog:
+
+- Rule Mining (market basket analysis)
+- Historical (common items)
```
Comment (finnqiao) [inline CHARTER.md]: Mention that the dataset involves historical purchases and thats what you're using. Rather than historical data of someone else or other unrelated things.

File: CHARTER.md [hunk 7]
```diff
@@ -0,0 +1,91 @@
...
+Icebox:
+
+- Optimize queries
+- Recommendation Feedback/Tracking
+
+### User Management
+
+Backlog:
+
+- Account Information (database)
```
Comment (finnqiao) [inline CHARTER.md]: Account information storage using what technology.

File: CHARTER.md [hunk 8]
```diff
@@ -0,0 +1,91 @@
...
+- New User
```
Comment (finnqiao) [inline CHARTER.md]: Make it clear that this is registration flow vs. data storage above. They seem to overlap.

File: CHARTER.md (hunk 2 above)
Comment (michaelfedell) [reply CHARTER.md]: that's 20 users total

File: CHARTER.md (hunk 1 above)
Comment (michaelfedell) [reply CHARTER.md]: Think I will actually pull the social part out of icebox and at least a minimally sufficient portion of recipe bit

File: CHARTER.md (hunk 4 above)
Comment (michaelfedell) [reply CHARTER.md]: Data is currently in CSV format and will be moved to a relational database for purpose of speed, persistence, etc.

File: CHARTER.md (hunk 6 above)
Comment (michaelfedell) [reply CHARTER.md]: This is actually historical trends for each user - e.g. recommendations will be driven by common rules (across universe of shoppers) as well as your own shopping history (Sally buys milk every week)

File: CHARTER.md (hunk 7 above)
Comment (michaelfedell) [reply CHARTER.md]: This will be in a relational database (likely postgres)
